config/300-daemon-containerd.yaml

```diff
@@ -27,11 +13,16 @@ spec:
         name: freeze-daemon
     spec:
       serviceAccountName: freeze-tokenreview # TODO: swap for account with just tokenreview
+      nodeSelector:
+        knative.dev/container-runtime: containerd
```

Comment:
this is super cool! Im wondering if we need something similar for the docker version to avoid both daemons ending up running on the same node?

Reply:
yeah, `daemon-docker` already have the same label